roles/container-engine/containerd/defaults/main.yml

```diff
@@ -8,6 +8,9 @@ containerd_systemd_dir: "/etc/systemd/system/containerd.service.d"
 containerd_oom_score: 0
 
 containerd_default_runtime: "runc"
+# Also you can dynamic set container runtime by ansible inventory group
+# containerd_default_runtime: "{{ 'nvidia' if inventory_hostname in groups['nvidia_gpu_nodes'] else 'runc' }}"
+
```

Comment:
This is wrong advice, this is the kind of settings which should go in a `group_vars/nvidia_gpu_nodes.yml` in your inventory instead.